src/hub.js

```diff
@@ -113,6 +114,13 @@ concurrentLoadDetector.start();
 store.update({ profile: { ...generateDefaultProfile(), ...(store.state.profile || {}) } });
 
 async function exitScene() {
+  if (NAF.connection.adapter && NAF.connection.adapter.localMediaStream) {
+    const tracks = NAF.connection.adapter.localMediaStream.getTracks();
```

Comment:
very minor but since this isnt perf sensitive code at all id use forEach
`NAF.connection.adapter.localMediaStream.getTracks().forEach(t => t.stop())`